translations/zh-hans.yaml

```diff
@@ -4253,7 +4253,6 @@ clusterNew:
       text: 编辑主机选项将更新主机注册命令
     command:
       instructions: '复制以下命令在主机的 SSH 终端运行。'
-      winInstructions: '在一台或者多台运行了受支持的 Docker 版本的主机上运行<code>CMD</code>命令，并且 windows server 版本大于<code class="code-version">{version}</code>.'
```

Comment:
we have this use case for it:
https://github.com/momesgin/ui/blob/master/lib/shared/addon/components/custom-command/template.hbs#L223
can we still remove it?

Reply:
It should fall back to the translation in the en-us file. Having arguments in other l18n files and not en-us caused some of the console errors in https://github.com/rancher/dashboard/issues/9361. In other cases I was able to update the l18n files to no longer use the argument in question, but with this line, the change in en-us was grammatically too complex for me to guess at how to replicate it.